apps/src/lib/kits/maker/LedMatrix.js

```diff
@@ -29,6 +29,10 @@ export default class LedMatrix {
   }
 
   toggle(x, y, brightness) {
-    return this.matrix[x][y] > 0 ? this.off(x, y) : this.on(x, y, brightness);
+    this.matrix[x][y] > 0 ? this.off(x, y) : this.on(x, y, brightness);
```

Comment:
Doesn't need to return anymore?

Reply:
Not necessary here - with what we've built out so far, it doesn't matter if we return anything here.